genai-perf/genai_perf/goodput_calculator/llm_goodput_calculator.py

```diff
@@ -45,7 +46,7 @@ class LLMGoodputCalculator(GoodputCalculator):
     def __init__(
         self,
         goodput_constraints: Dict[str, float],
-        metric: Union[LLMMetrics, Metrics],
+        metric: Union[LLMMetrics, ImageRetrievalMetrics, Metrics],
```

Comment:
can we set this to base class?
```suggestion
        metric: Metrics,
```